spec/associations/active_record_extensions_spec.rb

```diff
@@ -136,15 +136,17 @@ class Book < ActiveRecord::Base
 
         if ActiveRecord::VERSION::MAJOR > 3
           it "doesn't interfere with AR's procs in belongs_to methods" do
-            School.belongs_to :country, lambda { where() }
+            School.belongs_to :country, lambda { select(:id) }
```

Comment:
does this still work if someone puts a `where` here?

Reply:
@kbrock 

> I've only used a where scope in those instances. In your code, what do you put into the scope (that we were dropping and you added back?)

In our use case, we have a feature that automatically decrypts the contents when a column is loaded. That process can take a long time, and we are optimizing performance by skipping that process by not loading unnecessary columns. We use select to achieve this.

> does this still work if someone puts a where here?

Sure. I searched and thought it would be more common for use cases to use "where" in scope, so I modified the test to use "where" as well. What do you think? [20cb9bc](https://github.com/active-hash/active_hash/pull/255/commits/20cb9bcb96552d2c498c9c7f62e4443fd71f0985)

This is the search results.
https://github.com/search?l=Ruby&q=belongs_to%3A+lambda&type=Code